src/emailservice/Dockerfile

```diff
@@ -1,20 +1,19 @@
 FROM python:3-slim as base
 
 FROM base as builder
+FROM base as final
```

Comment:
@kalyanac I think this part doesn't make sense. We use "builder" below in line :25  as we install things in it. 

This should be like

```
FROM base as builder
COPY reqs.txt
RUN pip install

FROM base as final
RUN apt get
RUN install other stuff
COPY --from=builder
```

makes sense?

Reply:
my bad. Update the commit with fixes.